app/javascript/src/components/Dashboard/Contacts/utils.jsx

```diff
@@ -0,0 +1,129 @@
+import React from "react";
+
+import { MenuHorizontal } from "neetoicons";
+import { Dropdown, Avatar, Typography } from "neetoui";
+
+const renderNameRoleAvatar = (name, { role }) => (
+  <div className="flex space-x-4">
+    <Avatar
+      user={{
+        imageUrl: "https://i.pravatar.cc/300",
+      }}
+    />
+    <div>
+      <Typography style="h5">{name}</Typography>
+      <Typography className="neeto-ui-text-gray-600" style="body3">
+        {role}
+      </Typography>
+    </div>
+  </div>
+);
+
+const renderDropdown = () => (
```

Comment:
Be specific whenever possible. Here we are not rendering just any `dropdown`. It's an `ActionDropdown`.

Reply:
Right, I will take care of it.